logexp/README.md

```diff
@@ -0,0 +1,40 @@
+# LogExp
+
+LogExp is a tool that runs post-test on our kubernetes test clusters.
+It does the job of computing the set of logfiles to be exported (based on the
+node type (master/node), cloud provider, and the node's system services),and
+then actually exports them to the GCS path provided to it.
+
+The following flags need to be supplied to the tool:
+* --node-name (Name of the node this log exporter is running on)
```

Comment:
Explained it just to be similar with other flags. Removed the whole para anyway from the Readme now.